pkg/cloud/vsphere/actuators/cluster/actuator.go

```diff
@@ -72,6 +77,28 @@ func (a *Actuator) Reconcile(cluster *clusterv1.Cluster) (opErr error) {
 		return errors.Wrapf(err, "unable to reconcile certs while reconciling cluster %q", ctx)
 	}
 
+	// sync kubeconfig secrets for cluster
+	kubeConfig, err := a.GetKubeConfig(cluster, nil)
```

Comment:
Hi @andrewsykim,

Please consider renaming `kubeConfig` to `kubeConfigForTargetCluster` to make it super descriptive.

Reply:
Not sure if `kubeConfigForTargetCluster` is accurate here since this applies for any cluster represented by the cluster resource which also includes the management cluster

Reply:
Fair. I still think it's worth pushing this new code into a separate function (like ReconcileCertificates).

Reply:
Added a `reconcileKubeConfig` method